zone/zonedb.cpp

```diff
@@ -814,20 +814,24 @@ bool ZoneDatabase::LoadCharacterLeadershipAA(uint32 character_id, PlayerProfile_
 }
 
 bool ZoneDatabase::LoadCharacterDisciplines(uint32 character_id, PlayerProfile_Struct* pp){
-	std::string query = StringFormat(
-		"SELECT				  "
-		"disc_id			  "
-		"FROM				  "
+	auto query = fmt::format(
```

Comment:
Not suggesting it needs to be in this PR, but seems like a great opportunity to just use `CharacterDisciplinesRepository` here since its already made and available to clean this up and make it more readable

Reply:
Might as well do it now :) won't get done if not!